GCI-tixpro created a new branch and made changes

ineshtickoo opened the pull request:
Created a new branch because br1 was showing errors on pushing files.

f4ww4z commented on go.mod:
```diff
@@ -2,4 +2,9 @@ module github.com/ineshtickoo/hellogg
+	github.com/gin-gonic/contrib v0.0.0-20191209060500-d6e26eeaa607
+	github.com/gin-gonic/gin v1.5.0
+	github.com/gorilla/mux v1.7.3
+	golang.org/x/tools v0.0.0-20200103221440-774c71fcf114 // indirect
```
You don't need to make changes to this file for now. What is `mux` and `golang.org/x/tools` for?

ineshtickoo replied:
They were required for an earlier version of this code. Should I remove them? (I used mux for routing earlier)

f4ww4z replied:
> They were required for an earlier version of this code. Should I remove them? (I used mux for routing earlier)

Yes, we won't need them for now.

ineshtickoo commented:
> @ineshtickoo Please also squash your commits into 1. Your PR title should describe clearly what is changed in this PR.

Squashed the commits and renamed the PR.
